### DISCUSSION TITLE
Changing widget type for a layer doesnt update on the cloud. And new layers dont upload

### DISCUSSION THREAD

**Discussion by VilleBio** [opengisch/QField#3949] [Bugs]
I'm trying to change an existing layer widget in qgis on the pc from "time" to "string" but it doesn't work to get the widget changed on the cloud. This is already a created qfield cloud project

Also, I'm trying to upload new layers from a pc to an existing project on the qfieldcloud but I get an error on the mobile device saying it can't read the file/find it or something like that.

**Comment by lucienicolier** [comment]
Hi,

what kind of subscription do you have?

**Reply by VilleBio** [reply]
We have tried to get an organisation plan running. But had problems with the "pay" button after registration. So I guess we are still in beta?

**Reply by VilleBio** [reply]
Problem is now solved. It seemed to be an issue within qgis and not qfield. Thanks for replies.

**Reply by lucienicolier** [reply]
Nice to hear this.

Enjoy using QFieldCloud!

Best
